mozilla-mobile/android-components/components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -57,8 +57,8 @@
     <!-- The text shown inside the advanced button used to expand the advanced options. It's only shown when a website uses HSTS. -->
     <string name="mozac_browser_errorpages_security_bad_hsts_cert_advanced">Advanced…</string>
     <!-- The advanced certificate information shown when a website uses HSTS. The %1$s will be replaced by the website URL and %2$s will be replaced by the app name. It's only shown when a website uses HSTS. -->
-    <string name="mozac_browser_errorpages_security_bad_hsts_cert_techInfo"><![CDATA[
```

Comment:
@mcarare 
Thanks for making the change. Could you please confirm whether or not the old string be deleted or should it have `moz:RemovedIn=` and `tools:ignore=`?

Reply:
Going to add that also. Thank for pointing that out!

Reply:
@bcolsson Done!

Reply:
Thanks! Looks good to go.